examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -89,6 +96,7 @@ void ModelCommand::ReceiveCommandResponse(ChipDeviceController * dc, PacketBuffe
     uint8_t frameControl;
     uint8_t sequenceNumber;
     uint8_t commandId;
+    bool isGlobalCommand = false;
 
     // Bit 3 of the frame control byte set means direction is server to client.
```

Comment:
This comment shouldn't be here, and the comment about "Bit 3" a few lines down should move into `_isValidFrame` and be fixed to match the new reality.